playground-common/src/Debug/Trace/Extra.js

```diff
@@ -0,0 +1,14 @@
+/*eslint-env node*/
+'use strict';
+
+exports.traceTime = function () {
```

Comment:
Is this right? You seem to be returning an effect (zero arg function), but the FFI declaration doesn't show it.

Reply:
This is just like `Debug.trace`. It enables you to provide a thunk as the second argument.